docfx_yaml/extension.py

```diff
@@ -1428,19 +1438,26 @@ def convert_cross_references(content: str, current_object_name: str, known_uids:
     }
     known_uids.extend(hard_coded_references.keys())
 
+    # Used to keep track of current position to avoid converting if needed.
+    example_index = len(content)
     for index, word in enumerate(words):
+        if ignore_examples and example_text in word:
+            example_index = index
         uid = find_uid_to_convert(
             word, words, index, known_uids, current_object_name, processed_words, hard_coded_references
         )
 
-        if uid:
+        # If the reference is found after example section, ignore it.
+        if uid and (
+            not ignore_examples or
+            (ignore_examples and index < example_index)
```

Comment:
LGTM, as long as there cannot be content after the example.

Reply:
`Examples:` is only used by the bigframes team and it's their last content in the summary.